File: Sem4task27звездочка/program.py
num=input()         #Вводим строку
def sum_numbers_str(num):      #Создаю функцию на подсчет суммы символов
    total = 0       #создал переменную в которую буду вставлять сумму
    for i in range(len(num)): #создал цикл на длину строки
        total+=int(num[i])  #преобразовываю строку в число и суммирую все цифры
    return str(total)

print(sum_numbers_str(num))     #вывод нашей функции

File: Sem4task27звездочка/Program.cs
string ReadData(string msg)
{
    Console.WriteLine(msg);
    return (Console.ReadLine() ?? "0");
}

void PrintData(string msg1, string msg2)
{
    Console.WriteLine(msg1);
    Console.WriteLine(msg2);
}

string SumNumbers(string num)
{
    int total = 0;
    for (int i = 0; i < num.ToString().Length; i++)
    {
        int numbers = int.Parse(num);
        total += (num[i] - 48);

    }
    return total.ToString();
}

string number = ReadData("Введите число:");
string sum = SumNumbers(number);

PrintData("Сумма чисел равна: ", sum);





